spec/factories/pairings.rb

```diff
@@ -1,5 +1,6 @@
 FactoryBot.define do
   factory :pairing do
-    
+    time { 'lunch' }
+    date { 'Wed Apr 03 2020'}
```

Comment:
Note to self and others: if we need to have more dynamic test pairings I could rig this up to generate random times and dates in the same format. Time would be easier, something like this:
```ruby
time { ['morning', 'lunch', 'afternoon'].sample }
```